# sarscov2-brazil-seqdata
Exploring the sequencing data from SARSCoV2 patient SPBR1 in Brazil
Information about the case and sequencing copied from a post on [virological.org](http://virological.org/t/first-cases-of-coronavirus-disease-covid-19-in-brazil-south-america-2-genomes-3rd-march-2020/409): 

>To investigate the genomic diversity of these two cases, genomes were generated by the CADDE project at the IAL together with the Institute of Tropical Medicine team from the University of SÃ£o Paulo (IMT-USP). We used the openly-available COVID-19 sequencing and bioinformatics protocols developed by the ARTIC network 133. Sequencing protocol, sequencing multiplex primers, and bioinformatic protocols are described in detail in https://artic.network/ncov-2019 467. cDNA synthesis was conducted in duplicate for each sample. The concentration of PCR products was measured using a Qubit dsDNA High Sensitivity kit on a Qubit 3.0 fluorometer (ThermoFisher).

>Library preparation was conducted without a barcoding step and libraries were sequenced on an R9.4.1 flow cell. Sequencing was conducted in MinKNOW version 19.10.1 for over 12 hours. The open-source software RAMPART version 1.0.5 15 was used to assign and map reads in real-time. Raw files were basecalled with Guppy 5, demultiplexed and trimmed with Porechop 9 and mapped against reference strain Wuhan-Hu-1 (GenBank accession number MN908947 34). Variants were called using nanopolish 0.11.3 6 and accepted if they had a log-likelihood score of greater than 200. Low coverage regions were masked with N characters. Example of read data for SPBR1 is available for inspection from https://cadde.s3.climb.ac.uk/covid-19/BR1.sorted.bam 145. Consensus genome sequences have been deposited in GISAID with the following IDs: SPBR1: EPI_ISL_412964 and SPBR2: EPI_ISL_413016.
